Tests/test_imageops.py

```diff
@@ -94,6 +94,22 @@ def test_scale(self):
         newimg = ImageOps.scale(i, 0.5)
         self.assertEqual(newimg.size, (25, 25))
 
+    def test_colorize(self):
+        # Test the colorizing function
+
+        # Grab test image
+        i = hopper("L").resize((15, 16))
+
+        # Test original 2-color functionality
+        ImageOps.colorize(i, 'green', 'red')
+
+        # Test new three color functionality (cyanotype colors)
+        ImageOps.colorize(i,
+                          (32, 37, 79),
+                          (255, 255, 255),
+                          (35, 52, 121),
+                          40)
```

Comment:
Same here, but in a new test case using a fresh input image.

Reply:
do you mean i should split out the two tests (2 color, 3 color) into two different test methods in this class?

Reply:
i wasn't sure and i liked keeping them in one test, so i did that for now. let me know if i should change it though